README.md

```diff
@@ -11,153 +11,43 @@ its proof of work (POW). Taking development cues from Tenebrix and Litecoin,
-code. This is especially important for large or high-risk changes. It is useful
-to add a test plan to the pull request description if testing the changes is
-not straightforward.
-
-## Very Much Frequently Asked Questions ❓
-
-### How much doge can exist? – So many puppies! 🐕
-Early 2015 (approximately a year and a half after release) there were
-approximately 100,000,000,000 coins.
-Each subsequent block will grant 10,000 coins to encourage miners to continue to
-secure the network and make up for lost wallets on hard drives/phones/lost
-encryption passwords/etc.
-
-
-### Such mining information ⛏
-
-Dogecoin uses a simplified variant of the scrypt key derivation function as its
-proof of work with a target time of one minute per block and difficulty
-readjustment after every block. The block rewards are fixed and halve every
-100,000 blocks. Starting with the 600,000th block, a permanent reward of
-10,000 Dogecoin per block will be issued.  
-
-Originally, a different payout scheme was envisioned with block rewards being
-determined by taking the maximum reward as per the block schedule and applying
-the result of a Mersenne Twister pseudo-random number generator to arrive at a
-number between 0 and the maximum reward.
-
-This was changed starting with block 145,000, to prevent large pools from gaming
-the system and mining only high reward blocks. At the same time, the difficulty
-retargeting was also changed from four hours to once per block (every minute),
-implementing an algorithm courtesy of the DigiByte Coin development team, to
-lessen the impact of sudden increases and decreases of network hashing rate.
-
-**The current block reward schedule:**
-
-1–99,999: 0–1,000,000 Dogecoin
+## Ongoing development - Moon plan 🌒
 
-100,000–144,999: 0–500,000 Dogecoin
+Dogecoin Core is an open source and community driven software.  
+Development process is done publicly. Anyone can see, discuss and work on them.  
 
-145,000–199,999: 250,000 Dogecoin
+Main tools used for the core development :
 
-200,000–299,999: 125,000 Dogecoin
+* [Github Projects](https://github.com/dogecoin/dogecoin/projects) to see all developments and [Github Discussion](https://github.com/dogecoin/dogecoin/discussions) to discuss them
+* [Dogecoin Improvement Proposals (DIPs)](https://github.com/dogecoin/dips) for major improvements
+* [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
 
-300,000–399,999: 62,500 Dogecoin
+## Installation – omg developers 👨‍💻
 
-400,000–499,999: 31,250 Dogecoin
+To get all information to setup Dogecoin Core locally, see [INSTALL.md](INSTALL.md).
 
-500,000–599,999: 15,625 Dogecoin
+## Contribute 🤝
 
-600,000+: 10,000 Dogecoin
+Look at [CONTRIBUTING.md](CONTRIBUTING.md) to see how you can participate !
 
-**The original block reward schedule, with one-minute block targets and four-hour difficulty readjustment:**
+Do not limitate yourself to guidelines, feel free to contribute in your own way 🚀.
```

Comment:
Typo: `limit`

Reply:
Thx for report, will fix it when I go back into the code. I wait some feedbacks from maintainers (or anyone who want to give some :wink:) to do more work at once.

Reply:
Alright. I'll have a go later.

Reply:
@patricklodder @AbcSxyZ  What is the status on this?